storage/docs/graphql.md

```diff
@@ -1,45 +1,98 @@
 # GraphQL Api
 
-## /graphql endpoint
+## Data types
+
 
 ```
 type Tag {
   key: String!
   value: String!
 }
 
+input FileMetaInput {
+  key: String! # link from the service
+  namespace: String!
+  tags: [Tag]
+}
+
 type FileMeta {
-  id: ID!
```

Comment:
Why have we lost the ID ?

Reply:
replaced by 'key'

Reply:
Why don't we have both ... the `key` is not the same thing as `ID` ... at least not from an S3 perspective